src/modules/balance/ModuleTokens.vue

```diff
@@ -75,46 +103,47 @@ export default {
         })
         .map(item => {
           const newObj = {};
-          newObj.value = `$ ${item.usdBalance}`;
+          newObj.balance = [
+            new BigNumber(
+              this.web3.utils.fromWei(item.balance, 'ether')
```

Comment:
can you import fromWei from utils instead of importing web3? otherwise looks good

Reply:
done!